client/src/main/scala/drt/client/components/FlightTableComponents.scala

```diff
@@ -79,9 +86,9 @@ object FlightTableComponents {
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  def pcpTimeRange(arrival: Arrival, bestPax: (Arrival) => Int): TagOf[Div] = {
-    val sdateFrom = SDate(MilliDate(arrival.PcpTime))
-    val sdateTo = SDate(MilliDate(arrival.PcpTime + millisToDisembark(bestPax(arrival))))
+  def pcpTimeRange(arrival: Arrival, bestPax: Arrival => Int): TagOf[Div] = {
+    val sdateFrom = SDate(MilliDate(arrival.PcpTime.getOrElse(0L)))
```

Comment:
Having a 0 here should be impossible, but if we did fall back to a 0 it would definitely not make any sense... It would basically be saying that passengers start arriving at the Unix Epoch. Perhaps a better solution is to map over these options to get a value out, and then do a getOrElse(<.span())